src/PresentationalComponents/SystemsTable/SystemsTable.js

```diff
@@ -297,11 +301,12 @@ const SystemsTable = () => {
               workloads,
               dispatch,
               envContext.BASE_URL,
+              addNotification,
```

Comment:
downloadReport recieves addNotification as undefined as it needs to be 9th parameter, 8th is `display_name`
so it should be 
```
            downloadReport(
              'systems',
              fileType,
              filters,
              selectedTags,
              workloads,
              dispatch,
              envContext.BASE_URL,
              null,
              addNotification,
            ),
 ```
 otherwise we get `TypeError: addNotification is not a function DownloadHelper.js:54:14`